### ISSUE TITLE
Authenticator events

### ISSUE THREAD

**Issue by kurbaitis** [fenichelar/ember-simple-auth-token#209]
Hello everyone,

I'm trying to use JWT with refreshing of token each minute (for testing purposes). Ember `2.13.2`.

My API works correctly but I'm having problems with authenticator events, particularly `sessionDataUpdated` and `sessionDataInvalidated`. They just don't get fired. Well `sessionDataInvalidated` is a separate story because authenticator calls invalidate before triggering this event. But I need `sessionDataUpdated` very badly.

I tried to debug in the browser and it seems that from the JWT authenticator in https://github.com/jpadilla/ember-simple-auth-token/blob/master/addon/authenticators/jwt.js#L233 the trigger line is called but I can't attach to it in my instance initializer:
```
export function initialize(instance) {
  const applicationRoute = instance.lookup('route:application');
  const session          = instance.lookup('session:main');
  const authenticator    = instance.lookup('authenticator:jwt');

  session.on('authenticationSucceeded', function() {
    alert('authenticationSucceeded');

    applicationRoute._successfulSignIn();
  });

  session.on('invalidationSucceeded', function() {
    alert('invalidationSucceeded');

    applicationRoute._successfulSignOut();
  });

  authenticator.on('sessionDataUpdated', function (data) {
    alert('sessionDataUpdated');
  });

  authenticator.on('sessionDataInvalidated', function() {
    alert('sessionDataInvalidated');
    applicationRoute._successfulSignOut();
  });
};

export default {
  initialize,
  name:  'session-events',
  after: 'ember-simple-auth'
};
```

This code does not return any alerts from `sessionDataUpdated` and `sessionDataInvalidated`.

I'd appreciate any help or an idea how to attach to these events. Thanks

**Comment by jpadilla** [comment]
This might be fixed in master.
